### ISSUE TITLE
Add 1.0 and 1.1 test sites

### ISSUE THREAD

**Issue by jkotalik** [aspnet/AspNetCoreModule#190]
I had a misunderstanding of the point of the test sites before. For the out of process scenario, we should have test sites for 1.1 and 1.0 We can get the old 1.0 site from our git history, and make a new 1.1 test site. We then can add tests for each of the variations with a switch.

**Comment by shirhatti** [comment]
This is contingent on ANCM being built in Universe
@jkotalik Will set up a meeting

**Comment by Tratcher** [comment]
No, ANCM should have 1.0 and 1.1 regression test projects regardless of moving to universe.

**Comment by jkotalik** [comment]
Sorry to be clear, the location of these tests are contingent on ANCM being in Universe. We may want to move them to IISIntegration as it will contain all of the functional tests.

**Comment by muratg** [comment]
Any work remaining here?
